go.mod

```diff
@@ -34,6 +34,7 @@ replace (
 	github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
 	github.com/containers/buildah => github.com/containers/buildah v1.23.1
 	github.com/opencontainers/image-spec => github.com/opencontainers/image-spec v1.0.2-0.20210819154149-5ad6f50d6283
+	github.com/project-flotta/flotta-operator => github.com/ydayagi/k4e-operator v0.0.0-20220329090830-d066b756c24a
```

Comment:
what's missing in the operator main branch?

Reply:
my other PR :-)